app/server/lib/AttachmentStore.ts

```diff
@@ -76,6 +76,9 @@ export interface IAttachmentStore {
   // implementation and gives them control over local buffering.
   download(docPoolId: DocPoolId, fileId: FileId, outputStream: stream.Writable): Promise<void>;
 
+  // Remove attachment from the store
+  delete(docPoolId: DocPoolId, fileId: FileId): Promise<void>;
```

Comment:
This isn't actually used anywhere. I've left it in for now (since it's already implemented), but we could remove it?